app/lib/renderer/episode.rb

```diff
@@ -12,5 +12,21 @@ def render
       attach_images
       render_markdown
     end
+
+    # Extract vtt file
+    def generate_vtt_text
+      return if vtt.blank?
+
+      transcript = " ".dup
+      transcript = object.transcript.dup unless object.transcript.nil?
```

Comment:
I don't quite understand the purpose of this line?

Reply:
I created a new variable that ensures `object.transcript` always has a value and for when it is nil, it is assigned to the variable(empty string) that has `.dup ` method attached to it that fixes the string from being a frozen string literal.

Reply:
I mentioned this to you on Friday, but I think that we shouldn't be appending this to an existing value in the transcript field, but entirely replacing it. Otherwise, each time the video course gets an update, the transcript will get longer and longer?

Reply:
Hi @sammyd, Yes. This does make sense. I made some changes in the most recent commit.